components/pager/overview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Pager
+page_title: Pager for Blazor Overview
+description: Add a page navigation to Blazor application
+slug: components-pager-overview
+tags: telerik,blazor,pager,paging
+published: True
+position: 20
+---
+
+# Pager Overview
+
+The `Pager` component will enable you to add page navigation in your Blazor application. It follows the design patter of our components but as standalone one can be used outside of Grid.
+
+To use Telerik Pager component for Blazor:
```

Comment:
These instructions need to be shorter, drop the "optional" things. Add instructions when and where to provide the new set of data, and add a loop to render some items (even if it is just a few `<div>` elements with the ID of the item). This is the most important explanation on how to use the component and it needs to offer this information.

Reply:
The Localization demo is a good template for this

Reply:
Added demo